mlem/cli/serve.py

```diff
@@ -17,15 +18,15 @@
     config_arg,
     for_each_impl,
     lazy_class_docstring,
+    make_not_required,
 )
 from mlem.core.metadata import load_meta
 from mlem.core.objects import MlemModel
 from mlem.runtime.server import Server
 
 serve = Typer(
     name="serve",
-    help="""Deploy the model locally using a server implementation and expose its methods as
-endpoints.
+    help="""Create an API from model methods using a server implementation.
```

Comment:
"REST server" ?

Reply:
It depends on implementation. Some of them not even http-based